maraboupy/MarabouNetworkComposition.py

```diff
@@ -0,0 +1,233 @@
+'''
+Top contributors (to current version):
+    - Teruhiro Tagomori
+
+This file is part of the Marabou project.
+Copyright (c) 2017-2019 by the authors listed in the file AUTHORS
+in the top-level source directory) and their institutional affiliations.
+All rights reserved. See the file COPYING in the top-level source
+directory for licensing information.
+
+MarabouNetworkComposition represents neural networks with piecewise linear constraints derived from the ONNX format
+'''
+
+import numpy as np
+import onnx
+import sys
+import pathlib
+import os
+sys.path.insert(0, os.path.join(str(pathlib.Path(__file__).parent.absolute()), "../"))
+
+from maraboupy import Marabou
+from maraboupy import MarabouCore
+from maraboupy import MarabouNetwork
+from maraboupy import MarabouNetworkONNX
+
+class MarabouNetworkComposition(MarabouNetwork.MarabouNetwork):
+    """Constructs a MarabouNetworkComposition object from an ONNX file
```

Comment:
This comment could be more useful. We know that the object constructs a `MarabouNetworkComposition` object. What do objects of this class actually _represent_? What do they _do_?

Reply:
done